build.gradle

```diff
@@ -1,3 +1,8 @@
+plugins {
+  id "org.owasp.dependencycheck" version "1.4.5"
+  id "com.github.johnrengelman.shadow" version "1.2.4"
```

Comment:
I see shadow by default uses ```jfrog``` repo but we are publishing only to maven repo. Where are we specifying class path for owasp dependency check.